src/gate/parserGnet/parserGnet.h

```diff
@@ -0,0 +1,34 @@
+/**
+ * \brief Implements translation from Liberty format file to GNet.
+ * \author <a href="mailto:anushakov@edu.hse.ru">Aleksander Ushakov</a>
+ */
+
+#pragma once
+
+#include "gate/model/gnet.h"
+
+#include <kernel/yosys.h>
+
+#include <map>
+#include <memory>
+#include <vector>
+
+struct NetData {
+    std::vector<std::unique_ptr<eda::gate::model::GNet>> combNets;
+    std::vector<std::unique_ptr<eda::gate::model::GNet>> memNets;
+};
+
+void translateModuleToGNet(
+    const Yosys::RTLIL::Module *m,
+    NetData &net);
+
+void translateDesignToGNet(
+    const Yosys::RTLIL::Design &des,
+    NetData &vec);
+
+void translateLibertyToDesign(
+    const char* namefile,
+    NetData &vec);
+
+std::vector<uint64_t> truthTab(
+    const eda::gate::model::GNet *net);
```

Comment:
`truthTab` is a decent name for a variable, but it is not well-fitted for an API, consider to rename it

```suggestion
std::vector<uint64_t> buildTruthTable(
    const eda::gate::model::GNet *net);
```